app.js

```diff
@@ -0,0 +1,12 @@
+
+function info(name, breed, price) {
+  alert(name + ',' + ' ' + breed + ',' + ' ' + price);

```

Comment:
Would suggest: `alert(name + ', ' + breed + ', ' + price);` you can include the space with the comma. Also, a easier way is using string interpolation: `alert('Name: ${name} Breed: ${breed} Price: ${price}')` which allows you to do it all without the plus signs. Either works though!!